mysticeti-core/src/consensus/universal_committer.rs

```diff
@@ -59,7 +59,6 @@ impl UniversalCommitter {
 
                 // Try to directly decide the leader.
                 let mut status = committer.try_direct_decide(leader);
-                self.update_metrics(&status, Decision::Direct);
```

Comment:
why was this removed? should it be moved down to after `if status.is_decided()`

Reply:
That was erroneously put there to begin with. The `self.update_metrics` is called when the ordered leaders are returned , so that leads to a double count.